lib/components/tag-chip/style.scss

```diff
@@ -46,6 +53,10 @@
   .tag-chip {
     background: $studio-gray-70;
     color: $studio-white;
+
+    &.deleted {
+      background: $studio-red-70;
```

Comment:
@SylvesterWilmott For the dark theme, following the design guidelines, I used the dark version of the same color, let me know if I should use a different one.

## Screenshot
<img width="750" alt="tag-chip-deleted-dark" src="https://user-images.githubusercontent.com/14905380/114052943-c8687900-988e-11eb-965c-403515dc25b4.png">